static/common/css/fileindex.css

```diff
@@ -0,0 +1,70 @@
+body {
+    background-color: #222;
+    color: #ccc;
+}
+
+table {
+    border-collapse: separate;
+    border-radius: 6px;
+    margin-top: 2em;
+    min-width: 400px;
+    text-align: center;
+    width: 100%;
```

Comment:
With `width: 100%;`
![CSS_1](https://user-images.githubusercontent.com/57662928/126954550-3e1bcac6-1ef3-4647-8927-9c32108b6354.png)
Without `width: 100%;`
![CSS_2](https://user-images.githubusercontent.com/57662928/126954648-42336025-7fbc-4331-9507-decbb75488d7.png)
As for me, the second is prettier.

Reply:
Second. Fixed.